soft_introduction.html

```diff
@@ -162,19 +178,21 @@ <h1>Przyszłość</h1>
                 <section>
                     <img src="img/wysadz_ulice.png" style="background:none; border:none; box-shadow:none; "/>
                     <a href="http://wysadzulice.pl">http://wysadzulice.pl</a>
+                    <aside class="notes">
+                        <p> Wysadź ulice jest projektem wspołtworzonym z Koduj dla Polski. Jest to platforma, która ma na celu wspomaganie mieszkańców w projektowaniu i modernizacji ich najbliższego otoczeania. Dzięki tej stronie można zaprojektować otaczającą nas przetrzeń poprzez jej wizualizację. W prosty sposób można dodać ławkę do siedzenia czy klomb kwiatów.
```

Comment:
może udałoby się to zredagować i wypunktować zdania?
wydaję mi się, że tak byłoby łatwiej wejść z marszu i zacząć opowiadać

Reply:
@MargoWM projekt powstawał w ramach Koduj dla Polski, gdy ja współpracowałem z nimi. Oni zapewne chwalą się tą informacją na swojej stronie, no bo czemu mieliby tego nie robić, ale według mojej wiedzie od mojej rezygnacji ze współpracy z KdP nie widzę żadnej aktywności odnośnie tego projektu. To jest ich projekt do tego stopnia, że właścicielem domeny wysadzulice.pl jestem ja.

Może to lekka przesada, ale równie dobrze można powiedzieć, że KdP wysyła ludzi w kosmos, bo jeden z koordynatorów tego projektu zna kogoś, kto kupił sobie koszulkę NASA, gdy był na wymianie w Stanach.